doc/build.md

```diff
@@ -55,71 +61,85 @@ Orientation does not matter. Solder based on the labeled values.
-![USB port](https://i.imgur.com/MObHtyP.jpg)
+![USB port](https://imgur.com/2BZQcIB.png)
 
 ### Step 6. Capacitor (C5)
 **This part has a specific orientation** - longer leg goes in square pad (white mark on the capacitor points up)
-![Capacitor](https://i.imgur.com/ckbCSdM.jpg)
+![Capacitor](https://imgur.com/VPMZwld.png)
 
 ### Step 7. Fuse
 Orientation does not matter. Fold down after soldering.
-![Fuse](https://i.imgur.com/f7a9ctk.jpg)
+![Fuse](https://imgur.com/mGPtuYl.png)
 
 ### Step 8. Crystal (Y1)
 Orientation does not matter.
-![Crystal](https://i.imgur.com/ZtVHVa6.jpg)
+![Crystal](https://imgur.com/zMjaGQZ.png)
 
-### Step 9. LED
+### Step 9. LED (D71)
 **This part has a specific orientation** - shorter leg and flat side of the LED line up with the square pad
-![LED](https://i.imgur.com/6Xrkr5R.jpg)
+![LED](https://imgur.com/3Rwh3kY.png)
 
 ### Step 10. Push Button (RESET, BOOT) and IC Socket
 Push buttons have no specific orientation. \
 Align the notch on the IC Socket with the markings on the PCB. 
-![Push button and IC socket](https://i.imgur.com/4VjGMYQ.jpg)
+![Push button and IC socket](https://imgur.com/CIPhDrD.png)
 
 ### Step 11. Pin Header
 Orientation does not matter. 
-![Pin header](https://i.imgur.com/YeHUMEr.jpg)
+![Pin header](https://imgur.com/ZYlZZbF.png)
 
 ### Step 12. ATmega32A
-Insert ATmega32A into the IC socket. Make sure the notch on the microcontroller aligns with the notch in the IC socket and the markings on the PCB.
+Insert ATmega32A into the IC socket. Make sure the notch on the microcontroller aligns with the notch in the IC socket and the markings on the PCB then press gently.
+![ATmega32A](https://imgur.com/KjZF7Ug.png)
 
-### Step 13. M2 6mm Standoffs, M2 10+3mm Standoffs, M2 4mm screws
+### Step 12b (optional) : Flash your keyboard and check if everything is fine.
+It's better to check now before you solder anyhting else.
+
+### Step 13. M2 6mm Standoffs, M2 10+3mm Standoffs, M2 4mm screws, stabilizers
 | Component          | Qty | Location     |
 | ------------------ | --- | ------------ |
 | M2 10+3mm standoff | 6   | Front top    |
 | M2 4mm screws      | 5   | Front bottom |
 | M2 6mm standoff    | 11  | Back         |
-![Screws](https://i.imgur.com/ct2fOHf.jpg)
 
-### Step 14. Switches and Rotary Encoder
-Install the stabilizers, then place the plate on top. \
-Install and solder switches and rotary encoder.
+Now is a good time to insert stabilizers too. You can't insert standoffs or stabilizers once the switches have been soldered.
+![Screws](https://imgur.com/fUJYUYw.png)
+
+### 13b (optional) : Rotary encoders
+Install one or both the rotary encoders in the desired positions. Remember that you can only install one per side, top or bottom position for each. Solder one of the big legs while pressing it gently to ensure that it's flush. Solder the other leg and check. Solder the five pins.
```

Comment:
I'd include a note that it might be necessary to press the encoder pins with pliers from the sides to make the plate fit over the encoder - or did you not need to do that?